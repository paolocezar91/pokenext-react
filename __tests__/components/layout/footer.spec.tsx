import Footer from "@/components/layout/footer";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

describe("Footer", () => {
  it("should match snapshot with no valid description", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
