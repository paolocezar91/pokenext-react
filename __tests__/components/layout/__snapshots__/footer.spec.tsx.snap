// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot with no valid description 1`] = `
<DocumentFragment>
  <div
    class="   footer   container   w-full   fixed   bottom-0   flex   justify-between   border-solid   border-t-2   border-(--pokedex-red-light)   bg-(--pokedex-red)   py-2   px-4   "
  >
    <div
      class="w-full flex items-center justify-between text-xs text-white"
    >
      <p>
        By 
        <a
          class="underline"
          href="https://github.com/paolocezar91/"
          target="_blank"
        >
          PPC
        </a>
         with 
        <a
          class="underline"
          href="https://pokeapi.co/"
          target="_blank"
        >
          PokeAPI
        </a>
         and 
        <a
          class="underline"
          href="http://nextjs.org/"
          target="_blank"
        >
          Next.js
        </a>
        .
      </p>
      <div
        class="text-xs text-white"
      >
        <div>
          v0.4.1
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
